test/creddit/test/client.clj

```diff
@@ -119,6 +119,16 @@
     (is (thrown? Exception (client/subreddit-top creddit-client "aww" "10" :month)))
     (is (thrown? Exception (client/subreddit-top creddit-client "aww" 10 "month")))))
 
+(deftest test-subreddit-hot
+  (testing "Retrieve hottest subreddit posts"
+    (is (= (with-fake-routes
+             {"https://www.reddit.com/r/aww/hot/.json?limit=10"
+              (fn [request]
+                {:status 200 :headers {} :body reddit-response})}
+             (client/subreddit-hot creddit-client "aww" 10))
+           parsed-reddit-response))
+    (is (thrown? Exception (client/subreddit-hot creddit-client "aww" 10)))))
```

Comment:
I think this is the issue, this check is expecting an error to be thrown but the request is valid. Try changing `10` to `"10"` and it should pass as `"10"` is not a valid limit!

Reply:
You're right of course, passing now! Thanks very much for all the hand-holding :)

Reply:
No problem! Glad you go it working 😄